.ci/arduino.groovy

```diff
@@ -206,6 +211,7 @@ def buildEsp8266(config, sketches, String key) {
 					sketches[sketch].path != config.library_root+'examples/GatewaySerialRS485/GatewaySerialRS485.ino' &&
 					sketches[sketch].path != config.library_root+'examples/GatewayW5100/GatewayW5100.ino' &&
 					sketches[sketch].path != config.library_root+'examples/GatewayW5100MQTTClient/GatewayW5100MQTTClient.ino' &&
+					sketches[sketch].path != config.library_root+'examples/GatewayGSMMQTTClient/GatewayGSMMQTTClient.ino' &&
```

Comment:
Is your GSM gateway not supposed to be tested with at least one of the supported boards? In theory, I suppose it will work on any board with enough memory?

Reply:
I have no idea which hardware would be able to run it and which would not, so I just tried to follow the lead set by other MQTT gateway examples. However, looking at it right now it seems I followed the lead of ESP8266 example. Whoops.

I went back and only left tests disabled for the same hardware that was not allowing W5100MQTTClient examples.

Reply:
That sounds reasonable.